scheduler/src/swagger.ts

```diff
@@ -0,0 +1,31 @@
+import swaggerGen from 'swagger-jsdoc'
+
+const options: swaggerGen.Options = {
+  definition: {
+    info: {
+      title: 'Scheduler API',
+      version: '1.0.0',
+      description: 'API of the ODS Scheduler',
+    },
+    host: 'localhost:9000',
+    basePath: '/api/scheduler',
+  },
+  // path to files with swagger annotations
+  // Note: relative to package.json
+  // Note: since we transpile and then start, reference the dist directory + transpiled js files
+  apis: [
+    'dist/index.js', // REST API
+    'dist/interfaces/scheduling-job.js', // model
+    'dist/interfaces/datasource-config.js', // model
+  ],
+};
+
+// initialize swagger-jsdoc
+const swaggerSpec = swaggerGen(options);
+
+export function enableSwagger(app: any) {
```

Comment:
Is it possible to export the swaggerSpec and define the endpoint in index.ts? 
I think I would prefer having all the endpoints in one place, even the meta ones describing the API itself :D

Reply:
I'd rather pull all files with API into an `api` directory or by domain. Declaring everything in the `index.ts` is something that I consider as ok, but not on the long run. Especially as services grow (one file for config endpoints, one file for stateless functionality endpoints for example)

Reply:
So would renaming it to `swaggerApi.ts` be ok for you?

Reply:
Agree, I also think having an api directory would be the cleanest solution. So, yes, renaming would be fine!